static/app/components/events/contexts/device/getDeviceKnownDataDetails.tsx

```diff
@@ -90,7 +93,7 @@ export function getDeviceKnownDataDetails({
             ? formatMemory(memory_size, free_memory, usable_memory)
             : undefined,
       };
-    case DeviceKnownDataType.STORAGE:
+    case 'storage':
```

Comment:
why not use the `enum`?

Reply:
These values are not part of the enum exported in an event. Instead they are a special "known device value" that is processed differently: https://github.com/getsentry/sentry/blob/bdbd28e7f6bace994b30d859cfd699ef01b5f007/static/app/components/events/contexts/device/getDeviceKnownDataDetails.tsx#L11-L18

We don't want to put them in the enum because they don't need to be part of the event type definitions - these are not part of the relay spec, instead a special "hack" to get us to categorize parts of the enum together.

For example, `memory` represents the `memory_size, free_memory, usable_memory` fields in the enum.